app/client/src/routes/home.tsx

```diff
@@ -405,6 +405,9 @@ export function Home() {
   }
 
   if (content.status === 'success') {
+    const eqDataUrl =
+      content.data.services.eqDataApi || origin + window.location.pathname;
+
```

Comment:
Since it looks like we're using the absence of a value for the `eqDataApi` as a flag for a local environment, could we move the `origin` logic here? That way the port would be updated when accessing any machine on the local environment, not just `localhost`:
```suggestion
    const eqDataUrl =
      content.data.services.eqDataApi || `${window.location.protocol}//${window.location.hostname}:9090${window.location.pathname}`;

```

Reply:
Good call